test/unit_tests/test_Sources_correctness.jl

```diff
@@ -80,16 +83,16 @@ dist = (
     GammaPrimitiveParticleDistribution(FT(100), FT(0.1), FT(1)),
     ExponentialPrimitiveParticleDistribution(FT(1), FT(1)),
 )
-kernel = CoalescenceTensor((x, y) -> 5e-3 * (x + y), 1, FT(10))
+order = 1
+kernel = CoalescenceTensor((x, y) -> 5e-3 * (x + y), order, FT(10))
 NProgMoms = [nparams(d) for d in dist]
-r = kernel.r #maximum([ker.r for ker in kernel])
 thresholds = [FT(0.5), Inf]
 coal_data = initialize_coalescence_data(AnalyticalCoalStyle(), kernel, NProgMoms, dist_thresholds = thresholds)
 
 # action
 update_coal_ints!(AnalyticalCoalStyle(), dist, coal_data)
 
-n_mom = maximum(NProgMoms) + r
+n_mom = maximum(NProgMoms) + order + 1
```

Comment:
why is this one `order + 1` but in other places `r` is just replaced with `order`?